tests/test-cases/Group6-VIC-Machine/6-15-Syslog.robot

```diff
@@ -15,8 +15,8 @@
 *** Settings ***
 Documentation  Test 6-15 - Verify remote syslog
 Resource  ../../resources/Util.robot
-Suite Setup  Install VIC Appliance To Test Server  additional-args=--syslog-address tcp://%{SYSLOG_SERVER}:514 --debug 1
-Suite Teardown  Cleanup VIC Appliance On Test Server
+#Suite Setup  Install VIC Appliance To Test Server  additional-args=--syslog-address tcp://%{SYSLOG_SERVER}:514 --debug 1
+Test Teardown  Cleanup VIC Appliance On Test Server
```

Comment:
Just wonder what is difference between Suite and Test here?

Reply:
- Suite: execute only once for all test cases
- Test: execute each time for each test case

Reply:
I see. Thanks.